test/hisi_zip_test/test_lib.c

```diff
@@ -41,6 +42,239 @@ void *mmap_alloc(size_t len)
 	return p == MAP_FAILED ? NULL : p;
 }
 
+static void gen_random_data(void *buf, size_t len)
+{
+	int i;
+	uint32_t seed = 0;
+	unsigned short rand_state[3] = {(seed >> 16) & 0xffff, seed & 0xffff,
+					0x330e};
```

Comment:
code style problem: better to put longer line above the shorter line here ：）

xxxxx
xxx
xx